plugins/modules/python_requirements_info.py

```diff
@@ -23,8 +23,9 @@
     description: >
       A list of version-likes or module names to check for installation.
       Supported operators: <, >, <=, >=, or ==. The bare module name like
-      I(ansible), the module with a specific version like I(boto3==1.6.1), or a
-      partial version like I(requests>2) are all valid specifications.
+      I(ansible), the module with a specific version like I(boto3==1.6.1), a
```

Comment:
And while we're at it:
```suggestion
      C(ansible), the module with a specific version like C(boto3==1.6.1), a
```